gtsam/hybrid/tests/testGaussianMixtureFactor.cpp

```diff
@@ -191,8 +196,240 @@ TEST(GaussianMixtureFactor, Error) {
   DiscreteValues discreteValues;
   discreteValues[m1.first] = 1;
   EXPECT_DOUBLES_EQUAL(
-      4.0, mixtureFactor.error({continuousValues, discreteValues}),
-      1e-9);
+      4.0, mixtureFactor.error({continuousValues, discreteValues}), 1e-9);
+}
+
+/* ************************************************************************* */
+// Test components with differing means
```

Comment:
Can you add an extensive comment about this example ? Also, I thought we were going to focus on the HBN -> HFG conversion. Not opposed to showing specifying factors directly is equivalent but I don’t see that happening here.